src/ImGui.NET/ImGui.cs

```diff
@@ -38,6 +39,11 @@ public static void PushID(string id)
             ImGuiNative.igPushIDStr(id);
         }
 
+        public static void PushID(object id)

```

Comment:
Seems okay, although I think realistically you could just call GetHashCode yourself...

Reply:
> although I think realistically you could just call GetHashCode yourself
I think i'm not the only one using this technique for assigning unique IDs to same-name windows/controls, also this would replicate the `ImGui::PushID(void * id)` behaviour of C++ API